src/components/Toggle/Toggle.module.css

```diff
@@ -0,0 +1,85 @@
+@import '../../design-tokens/mixins.css';
+
+.toggle__button {
+  border: 0;
```

Comment:
Could we get a transparent border instead of no border? I'm concerned about whether this thing would show up in windows high contrast mode (where background colors become white but borders become 100% opacity black).

Reply:
Using a transparent border also helps prevents components giggling around between states (e.g., adding/removing that border can cause it to move around)